hack/jenkins/windows_integration_teardown.ps1

```diff
@@ -14,4 +14,12 @@
 
 $test_home="$env:HOMEDRIVE$env:HOMEPATH\minikube-integration"
 
+if ($driver == "docker") {
```

Comment:
the `==` should be an `-eq`

huge shoutout to powershell

Reply:
>:(

Done haha, good spot!